src/main/java/com/tg5/service/contract/EventPayload.java

```diff
@@ -12,4 +14,8 @@ public class EventPayload implements Serializable {
     private Long id;
 
     private String name;
+
+    private AccountType accountType;
+
+    private Location location;
```

Comment:
Location in Scanner, not in Event?

Reply:
The requirement requests that. thinking abroad each event has its location.  At the lecture session teachers, were saying the locations of scanner need to match its preplanned event